app/models/manageiq/providers/azure/cloud_manager/vm_or_template_shared.rb

```diff
@@ -13,6 +13,11 @@ def storage_acct_service(connection = nil)
     @storage_acct_service ||= ::Azure::Armrest::StorageAccountService.new(@connection)
   end
 
+  def image_service(connection = nil)
+    @connection ||= connection || ext_management_system.connect
+    @image_service ||= ::Azure::Armrest::Storage::ImageService.new(@connection)
+  end
+
```

Comment:
We have something like this in the refresh_helper_methods.rb. Could that be mixed in here instead? That has the advantage of locking down the api-version, too.

Reply:
@djberg96 while that is an interesting thought, I would be loathe to mix in all the other refresher-related methods from that file into this code.  Also since this PR is only adding the highlighted method, but we already have a couple of others as well, I'd probably rather add an issue tracking this, and pull all the "services" from both here and the refresh_helper_methods.rb into a separate mixin just defining the services.  Does that sound good?

Reply:
@jerryk55 Ok, let's refactor that in the future.